### ISSUE TITLE
Setting up icons for neo-tree.nvim

### ISSUE THREAD

**Issue by sherloach** [iconicFonts/iconic-fonts#97]
**Describe the bug**
I have installed Fira Code IF and set to my kitty terminal, but my neo-tree.nvim icons still not change, how can I get the neo-tree icons match like the image in README?

**Expected behavior**

**Screenshots**
<img width="496" alt="SCR-20240818-tnor" src="https://github.com/user-attachments/assets/53b12f2f-aa31-479d-9a88-d2ec85537ccc">

**Comment by mdSlash** [comment]
IconicFonts uses different icon codepoints, so Kitty falls back to NerdFonts for those icons.

We’ll release an update soon to make the codepoints compatible with NerdFonts, so you won't have to reconfigure anything.

In the meantime, you can reconfig [nvim-web-devicons](https://github.com/nvim-tree/nvim-web-devicons#setup) to use IconicFonts icons, but I don't recommend it, see #92

**Comment by sherloach** [comment]
So basically, the next release will automatically replace all the NerdFonts with IconicFonts without any configuration. The icons displayed within Telescope.nvim or any nvim plugins that need icons will also be replaced. Isn't it?

**Comment by mdSlash** [comment]
That's right, you don't need to do anything else. Just swap out the old `Fira Code IF` with the new version, and you're good to go

**Comment by sherloach** [comment]
Sweet! Thanks for the fantastic project!

**Comment by mdSlash** [comment]
Just a heads up, IconicFonts include icons from different packs, and some of them aren’t in NerdFonts. If you want different styles of icons, you'll need to reconfigure them manually. Otherwise, You’ll have the same Nerd Fonts iconstyle, but with the icons centered both vertically and horizontally
